testing/lttng-tools/APKBUILD

```diff
@@ -13,34 +13,32 @@ depends_dev="popt-dev libxml2-dev userspace-rcu-dev lttng-ust-dev kmod-dev
 makedepends="$depends_dev bash grep coreutils automake autoconf"
 install=""
 subpackages="$pkgname-dev $pkgname-doc"
-source="https://lttng.org/files/$pkgname/$pkgname-$pkgver.tar.bz2
-	musl-has-no-dlmopen.patch"
-
-builddir="$srcdir/$pkgname-$pkgver"
+source="https://lttng.org/files/$pkgname/$pkgname-$pkgver.tar.bz2"
+options="!check" # Tests fail
```

Comment:
then no reason to add `check()`

Reply:
```
============================================================================
Testsuite summary for lttng-tools 2.10.5
============================================================================
# TOTAL: 4742
# PASS:  2891
# SKIP:  1782
# XFAIL: 0
# FAIL:  65
# XPASS: 0
# ERROR: 4
```

Reply:
did you check why they fails?

Reply:
I'm not familiar with lttng-tools but looking at the test output I can't spot anything that would be useful

Reply:
@z3ntu btw how many tests fail in this version?

Reply:
```
============================================================================
Testsuite summary for lttng-tools 2.10.6
============================================================================
# TOTAL: 4756
# PASS:  2905
# SKIP:  1782
# XFAIL: 0
# FAIL:  65
# XPASS: 0
# ERROR: 4
```

Reply:
(4 more pass instead of fail when util-linux is added as checkdepends installed)